packages/web/src/common/store/lineup/sagas.js

```diff
@@ -53,6 +54,10 @@ function* filterDeletes(tracksMetadata, removeDeleted) {
   const isUSDCGatedContentEnabled = yield getFeatureEnabled(
     FeatureFlags.USDC_PURCHASES
   )
+  const allowedIds = yield remoteConfig
+    .getRemoteVar(StringKeys.EXPLORE_PREMIUM_ALLOWED_USER_IDS)
+    ?.split(',')
+    .map((id) => parseInt(id))
```

Comment:
Just wondering if it's going to be a pain to have to look up user ids instead of using handles?

Reply:
oo good call